build.gradle

```diff
@@ -149,12 +150,17 @@ ext.libraries = [
 dependencies {
   compile group: 'org.apache.logging.log4j', name: 'log4j-api', version: '2.13.3'
   compile group: 'org.apache.logging.log4j', name: 'log4j-to-slf4j', version: '2.13.3'
+  compile group: 'com.azure', name: 'azure-storage-blob', version: '12.6.1'
+  compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.11.0'
```

Comment:
there are no source code actually. i suppose for tools kind of stuff we might have it but for now it's not there

Reply:
Update the Gradle file